webpack.config.js

```diff
@@ -2,13 +2,16 @@ const path = require('path');
 const MiniCssExtractPlugin = require('mini-css-extract-plugin');
 const { CleanWebpackPlugin } = require('clean-webpack-plugin');
 
-const genConfig = (filename, { inlineCss } = {}) => ({
+const genConfig = (outputName, { inlineCss, inputName = 'index.js' } = {}) => ({
   output: {
     path: path.resolve(__dirname, 'dist'),
-    filename,
+    filename: outputName,
```

Comment:
```suggestion
    filename: outputFilename,
```
like, to don't make names shorter than webpack does🤷‍♀️

Reply:
Ok, why not, more semantic